docs/source/workflows/observe/observe-workflow-with-dynatrace.md

```diff
@@ -0,0 +1,118 @@
+* Configure your workflow (YAML) or Python script to send traces to the OTel collector.
+* Run the workflow and view traces within Dynatrace
+
+## Step 1: Dynatrace Account
+
+You will need access to your Dynatrace environment.  If you don’t have one you can simply [signup to get started](https://www.dynatrace.com/signup/)
+
+## Step 2: Dynatrace API Token
+
+Dynatrace APIs use token based authentication. To generate an access token:
+1. Go to Access tokens.
+1. Select Generate new token.
+1. Enter a name for your token.
+1. Select these [required scopes for the OTLP API](https://docs.dynatrace.com/docs/shortlink/otel-getstarted-otlpexport#authentication-export-to-activegate):
+    * `openTelemetryTrace.ingest`
+    * `metrics.ingest`
+    * `Logs.ingest`
+1. Select Generate token.
+1. Copy the generated token to the clipboard. Store the token in a password manager for future use and for the configuration below.
+
+## Step 3: Configure OTel Collector
+
+Configure an OTel Collector configuration file using an `otlphttp` exporter to the Dynatrace OTLP API as shown in the example below. Refer to the [Dynatrace documentation](https://docs.dynatrace.com/docs/shortlink/otel-collector-config) as required.
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+      http:
+        endpoint: 0.0.0.0:4318
+processors:
+  cumulativetodelta:
+exporters:
+  otlphttp:
+    endpoint: "https://<YOUR-DYNATRACE-ENVIRONMENT>.live.dynatrace.com/api/v2/otlp"
+    headers:
+      Authorization: "Api-Token <YOUR-DYNATRACE-TOKEN>"
+service:
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: []
+      exporters: [otlphttp]
+    metrics:
+      receivers: [otlp]
+      processors: [cumulativetodelta]
+      exporters: [otlphttp]
+    logs:
+      receivers: [otlp]
+      processors: []
+      exporters: [otlphttp]
+```
+
+## Step 4: Install and run your configured OTel Collector
+
+There are many ways to deploy an [OTel Collector](https://docs.dynatrace.com/docs/ingest-from/opentelemetry/collector/deployment) but for this example, an OTel Collector is created using Docker with the configuration from the previous step into a file named `otelcollectorconfig.yaml` with the [Dynatrace distribution of the OpenTelemetry Collector](https://docs.dynatrace.com/docs/ingest-from/opentelemetry/collector).  
+
+```bash
+docker run -d -v $(pwd)/otelcollectorconfig.yaml:/etc/otelcol/config.yaml \ 
+-p 4318:4318 \ 
+dynatrace/dynatrace-otel-collector:0.34.0 
+```
+
+Once running, the collector endpoint is: `http://localhost:4318`. 
+
+## Step 5: Install the NVIDIA NeMo Agent toolkit OpenTelemetry Subpackages
+
+```bash
+# Install all optional telemetry extras
+uv pip install -e '.[telemetry]'
+
+# Install specific telemetry extras
+uv pip install -e '.[opentelemetry]'
+```
```

Comment:
If Dynatrace only requires OTel, then only mention installing the `opentelemetry` group

Reply:
Yes for this one... I just reduced to telemetry and opentelemetryhttps://github.com/robertjahn/NeMo-Agent-Toolkit/blob/develop/docs/source/workflows/observe/observe-workflow-with-dynatrace.md?plain=1#L91-L100